circuit_definitions/src/circuit_definitions/base_layer/mod.rs

```diff
@@ -104,6 +114,9 @@ pub enum ZkSyncBaseLayerStorage<
     KeccakRoundFunction(T),
     Sha256RoundFunction(T),
     ECRecover(T),
+    ECAdd(T),
+    ECMul(T),
+    ECPairing(T),
```

Comment:
i think its best to keep the order of when the circuit was added, so they should be at the bottom of the list, it will break backwards compatibility with the old `numeric_circuit_type`s